VTFLib/VTFFormat.h

```diff
@@ -95,6 +95,7 @@ typedef enum tagVTFImageFormat
 	IMAGE_FORMAT_ATI2N,						//!<  = Red, Green BC5 compressed format - 8 bpp
 	IMAGE_FORMAT_ATI1N,						//!<  = Red BC4 compressed format - 4 bpp
 
+	IMAGE_FORMAT_BC6H = 69,					//!<  = Red, Green, Blue, BC6H compressed format - 8 bpp
```

Comment:
`IMAGE_FORMAT_BC6H` should be 71

Reply:
Cool, I'll change the rest of the enums / lists to reflect that also.